CHANGELOG.md

```diff
@@ -13,12 +13,22 @@
 * XCAssets: the `colorAliasName` and `imageAliasName` template parameters are now deprecated and will be removed in the next major release.  
   [David Jennes](https://github.com/djbe)
   [#614](https://github.com/SwiftGen/SwiftGen/pull/614)
+  
+### Deprecations
```

Comment:
Note: if we're adding a "deprecations" section, we may want to check the other entries and see if they need to be moved here. For example, the entry right above this is one.

Reply:
Good point.

Reply:
Worth mentioning in #621 so we don't forget to reorder/revisit the CHANGELOG before releasing